### DISCUSSION TITLE
Timestamps on output

### DISCUSSION THREAD

**Discussion by NebriBlackwing** [Blightmud/Blightmud#868] [Ideas]
Hello, 

I"m enjoying Blightmud, I haven't seen any options for toggling timestamps in the mud output. I'd love to see this feature added as it's usually one of the first that I toggle when working with any new client. My suggestion is to add a toggle for something along these lines:

[12:34] Page from 'Player2': 
[14:45] Page from
[14:46] You arrive in a dark forest..

etc etc, the only relevant output required is the timestamp in brackets. 

Thanks for considering my idea :)

**Comment by LiquidityC** [comment]
There is a plugin that does just this. I haven't looked at it for some time so if it doesn't work just ping me and I'll fix it.

https://github.com/Blightmud/blightmud-timestamp
